pypeit/wavecalib.py

```diff
@@ -179,10 +179,25 @@ def build_waveimg(self, tilts, slits, spat_flexure=None):
                 Image holding tilts
             slits (:class:`pypeit.slittrace.SlitTraceSet`):
             spat_flexure (float, optional):
+            spec_flexure (float, `numpy.ndarray`_, optional):
+                Spectral flexure correction in pixels. If a float,
+                the same spectral flexure correction will be applied
+                to all slits. If a numpy array, the length of the
+                array should be the same as the number of slits. The
+                value of each element is the spectral shift in pixels
+                to be applied to each slit.
 
         Returns:
             `numpy.ndarray`_: The wavelength image.
         """
```

Comment:
Why are we not similarly type checking the spatial flexure? The code is not asymmetric.